src/mirth.mth

```diff
@@ -38,18 +90,86 @@ def(compile!, Path +World -- +World,
         num-errors @ trace!
         " errors." trace-ln!
         1 posix-exit!,
+	"No errors." trace-ln!
+    )
+
+
+    dup entry-point match(
+    SOME -> 
+      "Codegen." trace-ln!
+      Name.search match(
+        SOME -> >Def .word? match(
+          SOME -> arrow swap output-file unwrap run-output-c99!,
+	  NONE -> drop expect!(F, "Could not find entry point")
+        ),
+        NONE -> drop expect!(F, "could not find entry point")
+      ),
```

Comment:
```suggestion
      >Name >Def .word? unwrap-or("could not find entry point" panic!)
      arrow swap output-file unwrap run-output-c99!,
```

Reply:
Actually, instead of using the arrow directly from the entry point word, we should be building an arrow with the correct type, like in `elab-target-c99!`, because it's necessary to typecheck the entry point. This would also fix the stack trace issue.